src/PhpSpec/CodeGenerator/Generator/ClassGenerator.php

```diff
@@ -54,7 +54,7 @@ protected function renderTemplate(ResourceInterface $resource, $filepath)
             '%name%'            => $resource->getName(),
             '%namespace%'       => $resource->getSrcNamespace(),
             '%namespace_block%' => '' !== $resource->getSrcNamespace()
-                                ?  sprintf("\n\nnamespace %s;", $resource->getSrcNamespace())
+                                ?  sprintf("%snamespace %s;", PHP_EOL.PHP_EOL, $resource->getSrcNamespace())
```

Comment:
-1 for this. Almost OSS projects are using LF newlines in their source code, not the OS newlines (as they cannot ship OS-specific source code). this would make things hard for any windows contributor as it would inject CRLF here (I don't know any big PHP project choosing CRLF as their consistent line endings)